GLTFSDK/CMakeLists.txt

```diff
@@ -12,9 +12,11 @@ file(GLOB schema_deps
     "${CMAKE_CURRENT_LIST_DIR}/schema/*"
 )
 
+find_program(POWERSHELL_PATH NAMES pwsh powershell NO_PACKAGE_ROOT_PATH NO_CMAKE_PATH NO_CMAKE_ENVIRONMENT_PATH NO_CMAKE_SYSTEM_PATH NO_CMAKE_FIND_ROOT_PATH)
```

Comment:
I think you need "PATHS" in here: https://cmake.org/cmake/help/latest/command/find_program.html

Why are these excludes needed?

Reply:
Without the excludes find_program returned /Applications/powershell.app/Contents/MacOS/PowerShell.sh, not /usr/local/bin/pwsh, on the macOS build agents. It wasn't clear why this was happening so I disabled all the default paths apart from (NO_)SYSTEM_ENVIRONMENT_PATH, which seemed to fix the problem.

Reply:
Ah I see those are all individual options.